src/main/java/org/spongepowered/common/mixin/core/item/inventory/MixinContainer.java

```diff
@@ -80,6 +81,11 @@
     @Shadow
     public abstract Slot getSlot(int slotId);
 
+    @Shadow
+    public ItemStack slotClick(int slotId, int dragType, ClickType clickTypeIn, EntityPlayer player) {
```

Comment:
I would, but then I wouldn't be able to call it [from here](https://github.com/SpongePowered/SpongeCommon/pull/1119/files#diff-b2176c9c962c5746dc0ea3a49c183c31R47).